packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -2492,7 +2596,7 @@ namespace Ifpack2 {
 
       inline
       void 
-      serialGemv(const local_ordinal_type &blocksize,
+      SerialGemv(const local_ordinal_type &blocksize,
                  const impl_scalar_type * const __restrict__ AA, 
```

Comment:
`__restrict__` is not a standard C++ keyword.  This may cause build warnings that break warnings-as-errors builds, depending on their settings.  It would be best to use a macro that only invokes the `__restrict__` keyword if you know that the compiler supports it.  Does Kokkos have such a macro?

Reply:
I think that they have one. However, I use this keyword in kokkoskernels and often find out it helps vectorization. At this moment, it does not generate warnings from there.

Reply:
@kyungjoo-kim We could just use the keyword through a macro, defined as `__restrict__` if the compiler supports that, and defined as nothing otherwise.

Reply:
done.